### PR TITLE
accumulated updates

### PR DESCRIPTION
**Pull request by YuelinXin**
This pull request contains all the accumulated minor updates in the past months

### CODE REVIEW

File: application/static/JS/basic.js [hunk 1]
```diff
@@ -61,16 +61,16 @@ addEventListener("scroll", function() {
         document.getElementById("down").style.transform = "rotate(180deg)";
         if(scrollTop > clientH)
         {
-            navStyle.backgroundColor = "inherit";
-            navStyle.color = "rgb(132, 78, 235)";
-            navStyle.boxShadow = "0px 0px 14px rgba(0, 0, 0, 0.3)";
+            navStyle.backgroundColor = "rgba(0, 0, 0, 0.8)";
+            navStyle.backdropFilter = "blur(20px)";
+            navStyle.webkitBackdropFilter = "blur(20px)";
```
Comment (YuelinXin) [inline application/static/JS/basic.js:66]: There are potential issues with this js styling in chromium-based browsers

**Review by YuelinXin — COMMENTED** [review]
Further testing needed to determine the problem mentioned in main.js

**Comment by YuelinXin** [comment]
@EdKenzo @Yuheng-SWJTU @Foreverythin

File: application/static/JS/basic.js (hunk 1 above)
Comment (zzh8241102) [reply application/static/JS/basic.js:66]: A good fix🚀

**Review by Foreverythin — APPROVED** [review]
Thanks for contribution.

**Pull request merged**